src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1743,44 +1200,59 @@ private class ACCT_IndividualAccounts_TEST {
     }
 
     /*********************************************************************************************************
-    * @description Update a household account's record type, verify system fields are unset.
-    */
+    @description 
+        Update a Household Account's Record Type to another Record Type 
+    verify:
+        The specific system fields should not be set.
+    **********************************************************************************************************/ 
     public static testMethod void updateHHAccountRecType() {
+        Boolean isPersonAccountEnabled = UTIL_Describe.isValidField('Account', 'isPersonAccount');
 
-        List<RecordType> rectypes = [SELECT Id FROM RecordType WHERE SobjectType = 'Account' AND IsActive = true];
-        if (rectypes.size() < 2) {
+        String soql = String.format(
+            'SELECT Id{0} FROM RecordType WHERE SobjectType = \'\'Account\'\' AND IsActive = true',
```

Comment:
out of curiosity, why did you use string.format with a parameter in the soql string, versus building it incrementally?  most code in NPSP would be:

soql = 'SELECT Id';
if (isPersonAccountEnabled)
   soql += ', isPersonType';
soql += ' FROM .....'